h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -677,6 +719,27 @@ public void validate(GLM glm) {
           throw new IllegalArgumentException("Missing values handling should be specified as an instance of " + MissingValuesHandling.class.getName());
         }
       }
+      if(_control_variables != null) {
+        if ((_interactions != null || _interaction_pairs != null)) {
+          glm.error("_control_variables", "Control variables option is not supported with interactions.");
+        }
+        for(String col: _control_variables){
+          Vec v = train().vec(col);
+          if (v == null) {
+            glm.error("_control_variables", "Control variable '"+col+"' is not in the frame.");
```

Comment:
IIRC there is some function that returns columns that are used for prediction. I would update the function and use it accordingly here. I think not updating the function might break some functionality in StackedEnsemble.

Reply:
https://github.com/h2oai/h2o-3/blob/master/h2o-algos/src/main/java/hex/glm/GLMModel.java#L2273